docs/build/build-guide.md

```diff
@@ -187,50 +196,16 @@ comparably minimal effort.
 
 #### Building Parachains with Cumulus
 
-Diving further into building parachains,
-[Cumulus](https://github.com/paritytech/polkadot-sdk/tree/master/cumulus) is another set of tools
-that aid in building a parachain-ready blockchain for Polkadot or Kusama. Cumulus utilizes FRAME and
-Substrate to create an easy way to build your first parachain. It ensures the chain follows the
-Parachain Protocol.
-
-:::info
-
-_Cumulus clouds are shaped like dots; together, they form an intricate, beautiful and functional
-system._
-
-:::
+[Cumulus](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/cumulus/index.html)
+transforms FRAME-based runtimes into Polkadot-compatible parachain runtimes, and Substrate-based
+nodes into Polkadot/Parachain-compatible nodes. In other words, Cumulus can enable a regular runtime
+to meet the requirements to have its state verified by Polkadot and take part in shared security.
```

Comment:
Adding more info on the specifics of Cumulus. Ref: https://substrate.stackexchange.com/questions/12/what-is-cumulus

It does not **transform** Substrate-based node into a Polkadot/Parachain node per se. It moves the blockchain runtime onto the collator nodes who produce blocks and send the PoVs to relay chain, instead of finding consensus within their own network.

```suggestion
[Cumulus](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/cumulus/index.html) is a set of tools that allows you to convert a blockchain developed using Substrate and FRAME into a Polkadot-compatible Parachain. More specifically, it provides libraries for all of the necessary parts of the Polkadot protocol necessary for Parachains to work, for example:

- Creating new parachain blocks via Collators
- Listening to the relay chain for updates
- Synchronizing upgrades between the parachain and relay chain

```